index.html

```diff
@@ -62,7 +62,7 @@ <h3>体験会 7/26(土）17:50～18:50</h3>
                 <div class="news-item">
                     <span class="date">2024/02/18</span>
                     <h3>会員様以外の方も、レンタルコートは可能です♪</h3>
-                    <p>レンタルラケット￥４４０/本 レンタルシューズ￥３３０ もございます。</p>
+                    <p>レンタルラケット￥５５０/本 レンタルシューズ￥３３０ もございます。</p>
```

Comment:
ラケットだけでなくシューズも価格を最新化しましょう

```suggestion
                    <p>レンタルラケット￥５５０/本 レンタルシューズ￥４４０ もございます。</p>
```